src/sentry/static/sentry/app/components/dataPrivacyRules/selectField.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from '@emotion/styled';
+// eslint import checks can't find types in the flow code.
+// eslint-disable-next-line import/named
```

Comment:
Is this a warning or error? For warnings I think we would prefer to leave them.

Reply:
it's an error: /Users/priscila/Projects/sentry/src/sentry/static/sentry/app/components/dataPrivacyRules/selectField.tsx

Reply:
its an error 
![image](https://user-images.githubusercontent.com/29228205/82887126-d697fa00-9f47-11ea-8b76-c03b02fca45a.png)